### ISSUE TITLE
"The upstream source code is not entirely Free" on F-Droid meaning ?

### ISSUE THREAD

**Issue by kineita** [opendocument-app/OpenDocument.droid#200]
I see you have a build on F-Droid few days, I am curious about what parts in your code is The upstream source code is not entirely Free. Many thanks

**Comment by TomTasche** [comment]
Unfortunately that's more a question to the F-Droid team rather than us. I was surprised to see that the app was updated on F-Droid recently! Last time I checked they were using a version from 2013... Anyway, here's an explanation of what "The upstream source code is not entirely Free" means in general: https://f-droid.org/en/docs/Anti-Features/#UpstreamNonFree

In our case I'd imagine that they are referring to our usage of [Firebase](https://firebase.google.com/).
